imblearn/_min_dependencies.py

```diff
@@ -18,6 +18,9 @@
 JOBLIB_MIN_VERSION = "0.11"
 THREADPOOLCTL_MIN_VERSION = "2.0.0"
 PYTEST_MIN_VERSION = "5.0.1"
+CUML_MIN_VERSION = "21.10"
```

Comment:
Just wanted to highlight this comment https://github.com/scikit-learn-contrib/imbalanced-learn/pull/858#issuecomment-973120901

I believe using cuML here requires 21.12. 21.12 is the current nightly, which is on track for release this week https://docs.rapids.ai/maintainers

Reply:
Yep, but I am kind of struggling to install `cudatoolkit` with `conda`. Locally, the package is available but it seems that this is not the case on Azure Pipeline. I am currently making sure that I have the latest version of `conda`.

This said, since there is no GPU on Azure, is cuML falling back on some sort of CPU computing or it will just fail?
In which case, I should investigate to find a free CI service for open source where I can get a GPU.